packages/scripts/generate-tarball.sh

```
@@ -0,0 +1,59 @@
+  exit 1
+fi
+
+# Fail on error
+set -e
+
+# Echo commands
+set -x
+
+aws ecr get-login-password --region $AWS_REGION | docker login --username AWS --password-stdin $ECR_REPO_URI
+
+GITHUB_SHA=$(git rev-parse --short HEAD)
```

Shellcheck (0.10.0)

[warning] GITHUB_SHA appears unused. Verify use (or export if used externally).

(SC2034)